database/migrations/Version20171007045806.php

```diff
@@ -14,7 +14,7 @@ public function up(Schema $schema)
     {
         $this->abortIf($this->connection->getDatabasePlatform()->getName() != 'mysql', 'Migration can only be executed safely on \'mysql\'.');
 
-        $this->addSql('CREATE UNIQUE INDEX UNIQ_F4E2474F5E237E06 ON albums (name)');
+        //$this->addSql('CREATE UNIQUE INDEX UNIQ_F4E2474F5E237E06 ON albums (name)');
```

Comment:
Is this a modification to a previous migration? I am little confused by this change

Reply:
Yes, i have to go back and edit some older migration to integrate some gedmo fields. As we inserted the base user on the migration area and not on the seed area. I made this changes to the migration in minimal way that old users may just get small bugs and it is inevitable